autopush/endpoint.py

```diff
@@ -728,7 +725,8 @@ def get(self, uaid=""):
         Retrieves the router type/data for a UAID.
 
         """
-        if not self._validate_auth(uaid):
+        test, _ = validate_uaid(uaid)
+        if not test or not self._validate_auth(uaid):
```

Comment:
I only call this twice, both within context of _validate_auth(), it would be better to move validate_uaid within _validate_auth()